src/networking.c

```diff
@@ -1602,29 +1602,93 @@ int _writeToClient(client *c, ssize_t *nwritten) {
             c->sentlen = 0;
         }
     } else {
-        clientReplyBlock *o = listNodeValue(listFirst(c->reply));
-        size_t objlen = o->used;
+        int iovcnt = listLength(c->reply);
+        if (iovcnt > 1) {
+            if (iovcnt > IOV_MAX) iovcnt = IOV_MAX;
```

Comment:
It seems that this line is redundant, `iovcnt` will be reset to 0 below.

Reply:
done.